.gitignore

```diff
@@ -0,0 +1,25 @@
+/Makefile
```

Comment:
Just make it `Makefile` (without the leading slash) and all the files named Makefile in the project will be ignored.